app/src/main/java/be/scri/fragments/AboutFragment.kt

```diff
@@ -168,6 +169,11 @@ class AboutFragment : Fragment() {
             ),
         )
 
+    private fun resetHints() {
+        HintUtils.resetHints(requireContext())
+        (activity as MainActivity).showHint("hint_shown_about", R.string.about_hint_message)
```

Comment:
Could u replace this string with the localized strings available in string.xml and do the same for the other fragments' hint messages?

Reply:
I've made the changes. Thanks for the review.